app/views/helpers/injected/AccordionHelper.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import org.apache.commons.lang3.StringEscapeUtils.escapeEcmaScript
+
+@this()
+@(id: String, category: String, label: String, content : Html)(implicit messages: Messages)
```

Comment:
Please remove unused `messages`
```suggestion
@(id: String, category: String, label: String, content : Html)
```